lib/abstract_format.ml

```diff
@@ -113,13 +113,16 @@ and type_t =
   | TyAnyMap of line_t
   | TyMap of line_t * type_assoc_t list
   | TyVar of line_t * string
+  | TyFunAny of line_t
+  | TyFunAnyArity of line_t * type_t * type_t
```

Comment:
I think it is enough to be `TyFunAnyArity of line_t * line_t * type_t` like [ExprRemoteCall](https://github.com/yutopp/obeam/blob/c1f65123e2474ff5cc2b221ebf72a8468c7a53b6/lib/abstract_format.ml#L66). What do you think about it?

Reply:
I selected `type_t` to clarify meaning of it, however `line_t` is also good if there have been already some examples which are using this interface. I'll fix it.

Reply:
Fixed b1c5479